dev/com.ibm.ws.security.oidc.client_fat.jaxrs/publish/files/serversettings/oidcRSClientAPITestConfig.xml

```diff
@@ -0,0 +1,71 @@
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+	<authFilter id="myAuthFilter">
+		<requestUrl
+			id="myRequestUrl"
+			urlPattern="helloworld"
+			matchType="contains" />
+	</authFilter>
+
+<!--
+	<authFilter id="badAuthFilter">
+		<requestUrl
+			id="badRequestUrl"
+			urlPattern="/BadHelloworld"
+			matchType="contains" />
+	</authFilter>
+-->
+
+	<openidConnectClient
+		id="client01"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		sharedKey="secret"
+		httpsRequired="false"
+		inboundPropagation="required"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		validationEndpointUrl="${special_validationEndpoint}"
+		validationMethod="${oAuthOidcRSValidationType}"
+		issuerIdentifier="http://localhost:${bvt.prop.security_1_HTTP_default}/${providerRoot}/endpoint/${providerSample},https://localhost:${bvt.prop.security_1_HTTP_default.secure}/${providerRoot}/endpoint/${providerSample},http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet"
+		jwkEndpointUrl="${oidcJWKValidationURL}"
+		signatureAlgorithm="${speical_sigAlg}"
```

Comment:
Fix spelling of `speical_sigAlg`

Reply:
again, will fix when I work on userinfo.